### ISSUE TITLE
Is it possible to integrate it with PHP

### ISSUE THREAD

**Issue by DebasisBehera123** [mozilla/http-observatory#395]
Hi,
I want to integrate it with WordPress. Is it possible?

**Comment by wolffe** [comment]
Yes, use cURL with PHP to create your tests. Run the rescan API first, then run the retrieval one after an hour or so. Use a CRON job to automate it.

**Comment by april** [comment]
Thanks for answering, @wolffe.  :)
